platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/async/DefaultAsyncConsumerActionExecutor.java

```diff
@@ -18,7 +18,6 @@
 import org.gradle.internal.concurrent.CompositeStoppable;
 import org.gradle.internal.concurrent.ExecutorFactory;
 import org.gradle.internal.concurrent.ManagedExecutor;
-import org.gradle.internal.concurrent.ServiceLifecycle;
```

Comment:
💭 We moved this class here, because it's only used in this project. It's a 10+ year old class that deals with async execution; at this point it's unlikely to suddenly start gaining wide use. If that happens, we can always move it back.